cg/meta/upload/mutacc.py

```diff
@@ -0,0 +1,195 @@
+"""
+    Module to upload cases to mutacc
+"""
+
+import os
+import logging
+from collections import namedtuple
+
+from cg.apps import scoutapi, mutacc_auto
+
+LOG = logging.getLogger(__name__)
+
+
+class UploadToMutaccAPI():
```

Comment:
I think this API might become easier to use if we name all non-public methods to '_methodname', what do you think?

Reply:
sure, sounds good!

Reply:
great!